components/search/list-renderer/workflow.js

```diff
@@ -0,0 +1,41 @@
+// node_modules
+import PropTypes from "prop-types";
+// components/search/list-renderer
+import AlternateAccessions from "../../alternate-accessions";
+import {
+  SearchListItemContent,
+  SearchListItemMain,
+  SearchListItemMeta,
+  SearchListItemQuality,
+  SearchListItemTitle,
+  SearchListItemType,
+  SearchListItemUniqueId,
+} from "./search-list-item";
+
+export default function Workflow({ item: workflow }) {
+  return (
+    <SearchListItemContent>
+      <SearchListItemMain>
+        <SearchListItemUniqueId>
+          <SearchListItemType item={workflow} />
+          {workflow.accession}
+        </SearchListItemUniqueId>
+        <SearchListItemTitle>{workflow.name}</SearchListItemTitle>
+        <SearchListItemMeta>
+          <div key="lab">{workflow.lab}</div>
```

Comment:
This shows the lab path, but it should show the lab title. The usual way is to change igvfd to embed the `@id` and `title` in the search results, like most other objects. I _guess_ that’s done through `embedded_with_frame`? But you can put that into the 1134 ticket.